strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/User.java

```diff
@@ -128,6 +140,7 @@ public String toString()
                           .add("enabled", isEnabled())
                           .add("salt", getSalt())
                           .add("roles", getRoles())
+                          .add("securityToken", StringUtils.isEmpty(getSecurityTokenKey()) ? "[EMPTY]" : "[SEECRET]")
```

Comment:
I'm not sure I follow this particular change... What... is the default value `[EMPTY]`, or what? Could you please explain this part?